apel/db/unloader.py

```diff
@@ -207,10 +211,14 @@ def _write_messages(self, record_type, table_name, query, ur):
         records = 0
         for batch in self._db.get_records(record_type, table_name, query=query, records_per_message=self.records_per_message):
             records += len(batch)
-            if ur:
-                self._write_xml(batch)
+
+            if table_name == 'DaySummaries':
+                self._write_yaml(batch)
             else:
-                self._write_apel(batch)
+                if ur:
+                    self._write_xml(batch)
+                else:
+                    self._write_apel(batch)
```

Comment:
I wonder if this would be better as

```
if table_name == 'DaySummaries':
   ...
elif ur:
   ...
else:
    ...
```

Reply:
Well. Actually, I didn't want to disturb or change the existing flow of the code. The changes I added will not impact the pervious code :). Hope that make sense ?